tests/rbd/test_rbd.py

```diff
@@ -31,7 +31,13 @@ def one_time_setup(node, rhbuild, branch: str) -> None:
     """
     try:
         node.exec_command(cmd="ls ceph")
+        out, err = node.exec_command(cmd="cd ceph; git branch")
```

Comment:
Instead of `git branch`, could you use `git rev-parse --abbrev-ref HEAD`? It would return only the branch name